src/olympia/devhub/templates/devhub/new-landing/components/dev_footer.html

```diff
@@ -0,0 +1,69 @@
+<footer class="DevHub-Footer content">
```

Comment:
this is just a renamed and then cut down version of /new-landing/components/footer.html - not sure why github's diff viewer hasn't realised that (it did for the `.less` file).

Reply:
In git, renames are not actually renames but moving content from the old file to a new one, and the fact that it's a rename is detected through an heuristic... which fails in some cases like this one.